fhir2-appnt-api/src/main/java/org/openmrs/module/fhirappnt/api/Impl/FhirBahmniAppointmentServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package org.openmrs.module.fhirappnt.api.Impl;
+
+import lombok.AccessLevel;
+import lombok.Setter;
+import org.openmrs.module.appointments.model.Appointment;
+import org.openmrs.module.fhir2.api.FhirAppointmentService;
+import org.openmrs.module.fhir2.api.dao.FhirAppointmentDao;
+import org.openmrs.module.fhir2.api.translators.AppointmentTranslator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Primary
+@Component
+@Transactional
+@Setter(AccessLevel.PACKAGE)
+public class FhirBahmniAppointmentServiceImpl implements FhirAppointmentService {
+
+    @Autowired
```

Comment:
If you have setters defined, I would put the `@Autowired` annotation on them, rather than relying on field annotation.